src/Api.php

```diff
@@ -0,0 +1,443 @@
+<?php
+
+namespace OpenFoodFacts;
+
+use OpenFoodFacts\Document;
```

Comment:
these 2 imports are unused, please remove it

Reply:
The 4 imports in this class are used, 
Document: line 203
Collection : line 183
LoggerInterface: line 112
NullLogger: line 115

Reply:
Document and Collection is in the same namespace, no import required

Reply:
It's true, i didn't see it like that.